styles/ollama/catppuccin.user.css

```diff
@@ -57,6 +57,106 @@
     @accent-color: @catppuccin[@@lookup][@@accent];
     @icon-filter: @catppuccin[@@lookup][@icon_filter];
     color-scheme: if(@lookup = latte, light, dark);
+// hljs import from https://github.com/catppuccin/userstyles/blob/195d5b63822209bf024cf91f655c60823ee0b5cc/styles/trinket/catppuccin.user.css#L83-L181
```

Comment:
You can do it like we do for chatgpt it is hljs
https://github.com/catppuccin/userstyles/blob/195d5b63822209bf024cf91f655c60823ee0b5cc/styles/chatgpt/catppuccin.user.css#L19
and
https://github.com/catppuccin/userstyles/blob/195d5b63822209bf024cf91f655c60823ee0b5cc/styles/chatgpt/catppuccin.user.css#L58-L83